### ISSUE TITLE
Audio jack switch with zsh

### ISSUE THREAD

**Issue by ViktorWalter** [stockmind/gpd-pocket-ubuntu-respin#107]
For users who use zsh as the primary shell, the [Out] in the headphone-jack.sh script needs to be escaped in order to work. I have solved this for myself by removing the [Out] from the sink variables and putting it to the final command as `"pacmd set-sink-port alsa_output.platform-cht-bsw-rt5645.HiFi__hw_chtrt5645__sink \"[Out] $sink\""`.

**Comment by stockmind** [comment]
Hi, thanks for the tip! I also use zsh as my primary shell, but script is launched with bash shebang, so shouldn't be run under bash? there is something i'm missing?

Also, just to know, what kernel version are you using?

**Comment by ViktorWalter** [comment]
Yeah, the `#!/bin/bash` would normally pass everything to bash, but later on the command for output device selection is passed to `su "$user"` which invokes the user's default shell again.
My kernel is the current 4.16.0-rc3, updated yesterday through your script.
Also I have corrected my previous message - I forgot to set the command to code, so the escaping was invisible.

**Comment by stockmind** [comment]
Great! sorry for the mistake. 
What about call the command and bind it to /bin/bash? To let the command work independently
from the main shell used?
Something like:

```
su "$user" -s /bin/bash -c "pacmd set-sink-port alsa_output.platform-cht-bsw-rt5645.HiFi__hw_chtrt5645__sink $sink"
```
